### PR TITLE
React Stopwatch

### PR DESCRIPTION
**Pull request by Garcia-Tony**
https://github.com/user-attachments/assets/3a280d96-9efe-45c9-bd7d-6d180ff083e2

### CODE REVIEW

File: react-stopwatch/my-app/src/stopwatch.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      clearInterval(play);
+    }
+    setTime(0);
+    setPlay(false);
+  };
+
+  return (
+    <div>
+      <div className="face-input" onClick={reset}>
+        {' '}
```
Comment (dseongdev) [inline react-stopwatch/my-app/src/stopwatch.tsx]: what's the empty string for?

File: react-stopwatch/my-app/src/stopwatch.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        {time}{' '}
```
Comment (dseongdev) [inline react-stopwatch/my-app/src/stopwatch.tsx]: what's the empty string for?

File: react-stopwatch/my-app/src/stopwatch.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      </div>
+
+      <button className="time-input" onClick={playPause}>
+        {' '}
```
Comment (dseongdev) [inline react-stopwatch/my-app/src/stopwatch.tsx]: what's the empty string for?

**Review by dseongdev — CHANGES_REQUESTED** [review]
left comments

**Comment by Garcia-Tony** [comment]
idk why those empty strings were there, they dont do anything at all, fixed up and removed

**Review by dseongdev — APPROVED** [review]

**Pull request merged**